docs/api-guide.md

```diff
@@ -250,6 +250,36 @@ You can get table metadata as follows:
 TableMetadata tableMetadata = admin.getTableMetadata("ns", "tbl");
 ```
 
+### Repair a namespace
+
+If a namespace is in an unknown state such as the namespace exists in the underlying storage but not its ScalarDB metadata or vice versa.
+This method will recreate the namespace and its metadata if necessary.
+
+You can repair the namespace as follows:
+
+```java
+// Repair the namespace "ns" with options.
+Map<String, String> options = ...;
+admin.repairNamespace("ns", options);
+```
+
+### Repair a table
```

Comment:
Even though the Admin API `repairTable()` method has existed since v3.6. A section in the api-guide and storage-abstraction docs was never added. Since the content will be different for the older doc versions since the method behavior is changing for the next release, I will create another PR for it.